### DISCUSSION TITLE
Removing XP

### DISCUSSION THREAD

**Discussion by PerryRylance** [cjmellor/level-up#82] [Q&A]
The documentation mentions that the experience system can be turned off by adjusting the config, but it's unclear to me how to do this.

We don't need it for our project, how do I do this please?

**Comment by cjmellor** [comment]
@PerryRylance Does it? Can you point me to where it says that? I would imagine it is just in the config file?

**Comment by PerryRylance** [comment]
@cjmellor Says it here in the docs, but I couldn't see how to do this in the config or by looking at the source, sorry!

![image](https://github.com/cjmellor/level-up/assets/14136738/cbe8116b-5e18-4815-aff5-7ed0bbba352c)

**Comment by cjmellor** [comment]
I've removed it because I don't think that is the case... not sure why I added it to be honest as you're right, there is no code related to disabling it. In fact, it's pointless disabling it as it is a key part of the package...

Only

- Multiplier
- Level Cap
- Audit

Can be toggled on/off

Sorry about that.
